espnet2/tts/fastspeech/fastspeech.py

```diff
@@ -123,6 +65,7 @@ def __init__(
         postnet_layers: int = 5,
         postnet_chans: int = 512,
         postnet_filts: int = 5,
+        postnet_dropout_rate: float = 0.5,
```

Comment:
Regarding the change of arguments order: this may break user code. Assuming this class is used everywhere with kwargs (not args), this change would be okay.

Reply:
The same applies to Tacotron2 and Transformer.

Reply:
In espnet2, all model arguments are dealt as kwargs. Therefore is should be no problem.